code/objects/News.php

```diff
@@ -321,17 +321,18 @@ public function getMonthCreated(){
 		return $this->dbObject('PublishFrom')->Format('F');
 	}
 
-        /**
-         * Create a date-string based on the locale. Looks better.
-         * @return string
-         */
-        public function getPublished() {
-		i18n::get_date_format();
+	/**
+	 * Create a date-string based on the locale. Looks better.
+	 * @return string
+	 */
+	public function getPublished()
+	{
+		$format = i18n::get_date_format();
 		$locale = i18n::get_locale();
 		$date = new Zend_Date();
 		$date->set($this->PublishFrom, null, $locale);
-		return substr($date->getDate($locale),0,-9);
```

Comment:
Won't this break other dateformats? Wouldn't it be better to run a substring to the expected length? Or even better, I think this should be handled in template i18n-formatting?
(Ow, and thanks for formatting :) )